src/local.js

```diff
@@ -1,18 +1,61 @@
 'use strict';
 
-const n = require('numeric');
 const regression = require('./regression');
 const numeric = require('numeric');
 const get = require('lodash/get');
 const fs = require('fs');
 const FreeSurfer = require('freesurfer-parser');
+const pify = require('pify');
+
+/**
+ * Add bias.
+ *
+ * @param {?} A
+ * @returns {Array}
+ */
+function addBias(arr) {
+  if (!Array.isArray(arr) || !arr.length) {
+    throw new Error(`Expected ${arr} to be an array with length`);
+  } else if (!arr.every(Array.isArray)) {
+    throw new Error(`Expected every item of ${arr} to be an array`);
+  }
+
+  return arr.map(row => row.concat(1));
```

Comment:
This is _literally_ all `addBias` does, as far as I understand. Old test passes. No need to mutate arrays as matrices, just add the damn number.